lib/discuss/auth.ex

```diff
@@ -45,8 +46,11 @@ defmodule Discuss.Auth do
       iex> get_user_by(email: "some_user@email.com")
       %User{}
 
+      iex> get_user_by(email: "other@email.com")
+      nil
+
   """
-  def get_user_by([{attr, value}]) when is_atom(attr), do: Repo.get_by(User, attr, value)
+  def get_user_by(keyword_list), do: Repo.get_by(User, keyword_list)
```

Comment:
Eu sempre olho na própria documentação do ecto para saber qual nome dar ao arg quando estou passando para alguma função do Repo.

![Screenshot from 2022-01-19 11-30-33](https://user-images.githubusercontent.com/6487206/150150956-816f64ae-4824-4e59-9e82-de207c0afdd8.png)

```suggestion
  def get_user_by(clauses), do: Repo.get_by(User, clauses)
```

Acho que fica mais intuitivo